src/cli.js

```diff
@@ -59,7 +59,7 @@ program
   .option(
     '--log-level <level>',
     'set the log level, either "0" (no output), "1" (errors), "2" (warnings + errors) or "3" (all).',
```

Comment:
Should we update this message to add the new verbose modes?

Reply:
Probably, updated it to include the verbose and debug mode